rest/blip_sync.go

```diff
@@ -171,7 +193,7 @@ func (bh *blipHandler) handleGetCheckpoint(rq *blip.Message) error {
 		return err
 	}
 	if value == nil {
-		return base.HTTPErrorf(401, "Not found")
+		return base.HTTPErrorf(404, "Not found")
```

Comment:
Are we sure the client isn't expecting a 401 for a missing checkpoint?

Reply:
Also, if we're cleaning this up, should we replace the '404' with http.StatusNotFound?

Reply:
I filed https://github.com/couchbase/couchbase-lite-core/issues/384, and will roll it back out of this PR until we get confirmation